### DISCUSSION TITLE
Interval Location updates

### DISCUSSION THREAD

**Discussion by Ryansurjadi** [HaylLtd/cordova-plugin-background-geolocation#147] [Q&A]
Hi, i have an question about this plugin setup.
i want to track our user for every 5 minutes, how to config ? 
for now I'm using this config and tracker send me data every milisecond :
 stationaryRadius: 350,
 distanceFilter: 750,
 interval: 300000,
 fastestInterval: 180000,
 activitiesInterval: 180000
 
 i'm using http post to catch the data. I already read the documentation but not sure how is the plugin works.
 can anyone help me to explain this to me ?
 
 Thanks

**Comment by HarelM** [comment]
This looks OK I think, maybe set the fastetInterval and activitiesInterval to 300000 as well?
What are you getting in your server?
`stationaryRadius` of 350 would mean that if the user doesn't move more than 350 meters you won't get a GPS call, not matter how much time you set it, so I'm not sure this is what you want.
But I would say the best approach is to use trail and error and see which configurations works the best for you.
You can also use debug mode to hear when a new GPS position is sent.
